### PR TITLE
MBMS-77352 Added NCA error emails to email notifications

### PR DESCRIPTION
**Pull request by ksantiagoBAH**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

Added error email for NCA form to Notifications email

## Related issue(s)

https://jira.devops.va.gov/browse/MBMS-77352

## Testing done

- [x] *New code is covered by unit tests*
- *Describe what the old behavior was prior to the change*
- *Describe the steps required to verify your changes are working as expected. Exclusively stating 'Specs run' is NOT acceptable as appropriate testing*
- *If this work is behind a flipper:*
  - *Tests need to be written for both the flipper on and flipper off scenarios. [Docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Backendexample).*
  - *What is the testing plan for rolling out the feature?*

## Screenshots
_Note: Optional_

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [ ]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by jordanBoardmanCodes — DISMISSED** [review]

**Review by ConnorJeff — DISMISSED** [review]

File: modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb [hunk 1]
```diff
@@ -183,8 +191,11 @@ def get_first_name_from_form_data
         form20_10207_contact_info[1]
       when 'vba_40_0247'
         form_data.dig('applicant_full_name', 'first')
+      when 'vba_40_10007'
+        form_data.dig('application', 'claimant', 'name', 'first')
```
Comment (rmtolmach) [inline modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb:195]: @ksantiagoBAH Sorry if I missed it, but is this being tested anywhere?

Comment (ksantiagoBAH) [reply modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb:195]: I have some test in place for the notification email for the 40-10007, do you know where in the other lines of code above are being tested?

Comment (ksantiagoBAH) [reply modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb:195]: NVM

**Review by pennja — APPROVED** [review]
Looks good, great work! 👍

**Review by adam-antonioli — APPROVED** [review]

**Review by rmtolmach — APPROVED** [review]
thanks for adding that test!

**Pull request merged**